src/api-reference/invoice/purchase-request-endpoints.markdown

```diff
@@ -1,21 +1,44 @@
 ---
-title: Purchase Request
+title: Purchase Request v4
 layout: reference
 ---
 
 {% include prerelease.html %}
 
-# Purchase Request
+The Purchase Request API gives clients the ability to leverage external data to create Concur Invoice purchase requests for pre-authorization. Clients and Partners can build a direct connection where data can be fed to the purchase request API which will create new purchase requests in Concur. These API-created purchase requests are automatically submitted into the Concur Invoice pre-authorization workflow. Once approved, it will result in a Concur Invoice purchase order that can be transmitted to your Vendor from Concur Invoice. 
+In addition a Get Status endpoint is provided that can be used to get basic information and check the status of the created purchase request. 
 
+> **Limitations**: This API does not support users in the China data center.
+
+* [Products and Editions](#products-editions)
+* [Scope Usage](#scope-usage)
+* [Dependencies](#dependencies)
+* [Access Token Usage](#access-token-usage)
 * [Create a new purchase request](#post)
 * [Get details of purchase request](#get)
 * [Schema](#create_purchase_request_schema)
 * [Response schema](#create_purchase_request_schema-response)
 * [Error codes](#error-codes)
 
-## Version
+## <a name="products-editions"></a>Products and Editions
+
+* Concur Invoice Professional Edition
+* Concur Invoice Standard Edition
+
+## <a name="scope-usage"></a>Scope Usage
+
+Name|Description|Endpoint
+---|---|---
+`PurchaseRequest.Read`|Read only access to purchase request data|Get details of a purchase request
+`PurchaseRequest.Write`|Read and Write access to purchase request data|Create a new purchase request
+
+## <a name="dependencies"></a>Dependencies
+
+No other SAP Concur APIs are required to use this API. 
```

Comment:
Might I suggest simply saying 'None' here is more elegant than an entire sentence? Also, dependencies can take forms other than APIs, such as code libraries.

```suggestion
None
```